[Blueprints] Include change notes while publishing blueprint

filizt opened the pull request:
This is a breaking change. Please let me know if there is anything extra I should be doing for that.

Commit:
add support for including change notes while publishing blueprint

erich-wang commented:
@filizt , please
1. Provide swagger spec PR for this change
2. Fix CI build error
3. Follow [instruction](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#signing-and-publishing-nuget-packges) to update version if needed

Commit:
Update custimizations to support signature changes

Commit:
Update version number

filizt commented:
> @filizt , please
> 
> 1. Provide swagger spec PR for this change
> 2. Fix CI build error
> 3. Follow [instruction](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#signing-and-publishing-nuget-packges) to update version if needed

@erich-wang  -

1. Link for the swagger spec PR: https://github.com/Azure/azure-rest-api-specs/pull/6399
2. Fixed CI errors
3. Updated version

Let me know if there is anything else blocking the merge.
